api/v1/storageoscluster_types.go

```diff
@@ -180,7 +180,7 @@ type ContainerImages struct {
 	NFSContainer                       string `json:"nfsContainer,omitempty"`
 	APIManagerContainer                string `json:"apiManagerContainer,omitempty"`
 	NodeManagerContainer               string `json:"nodeManagerContainer,omitempty"`
-	UpgradeGuardContainer              string `json:"upgradeGuardContainer,omitempty"`
+	NodeGuardContainer                 string `json:"nodeGuardContainer,omitempty"`
```

Comment:
How we would do migration from old version to new, because this change breaks compatibility?

Reply:
Good question. I will create a ticket for this as I think it needs some more thought and is not a priority for now